services/minespace-web/src/tests/components/Forms/projects/projectSummary/__snapshots__/DocumentUpload.spec.js.snap

```diff
@@ -1,155 +1,42 @@
-          ".ott": "application/vnd.oasis.opendocument.text-template",
-          ".pdf": "application/pdf",
-          ".png": "image/png",
-          ".rtf": "application/rtf",
-          ".tif": "image/tiff",
-          ".tiff": "image/tiff",
-          ".xls": "application/vnd.ms-excel",
-          ".xlsx": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
-        }
-      }
-      component={[Function]}
-      id="documents"
-      name="documents"
-      onFileLoad={[Function]}
-      onRemoveFile={[Function]}
-      params={
-        Object {
-          "mineGuid": undefined,
-          "projectGuid": undefined,
-          "projectSummaryGuid": undefined,
-        }
-      }
-      props={
-        Object {
-          "documents": Object {},
-          "label": "Upload Files  (optional)",
-          "labelIdle": "<strong>Drag & Drop your files or <span class=\\"filepond--label-action\\">Browse</span></strong><br><div>We accept most common document, image, and spreadsheet with max individual file size of 400 MB.</div>",
-        }
-      }
-    />
-    <Connect(DocumentTable)
-      documentColumns={
-        Array [
-          Object {
-            "dataIndex": "document_name",
-            "key": "document_name",
-            "render": [Function],
-            "sorter": [Function],
-            "title": "File Name",
-          },
-          Object {
-            "className": undefined,
-            "dataIndex": "category",
-            "key": "category",
-            "render": [Function],
-            "title": "Document Category",
-          },
-          Object {
-            "dataIndex": "upload_date",
-            "key": "upload_date",
-            "render": [Function],
-            "sorter": [Function],
-            "title": "Updated",
-          },
-          Object {
-            "dataIndex": "create_user",
-            "key": "create_user",
-            "render": [Function],
-            "sorter": [Function],
-            "title": "Updated By",
-          },
-        ]
-      }
-      documentParent="project summary"
-      documents={Array []}
-    />
-  </InternalFormItem>
-</Fragment>
+<ContextProvider
+  value={
+    Object {
+      "store": Object {
+        "@@observable": [Function],
+        "dispatch": [Function],
+        "getState": [Function],
+        "replaceReducer": [Function],
+        "subscribe": [Function],
+      },
+      "subscription": Subscription {
+        "handleChangeWrapper": [Function],
+        "listeners": Object {
+          "notify": [Function],
+        },
+        "onStateChange": [Function],
+        "parentSub": undefined,
+        "store": Object {
+          "@@observable": [Function],
+          "dispatch": [Function],
+          "getState": [Function],
+          "replaceReducer": [Function],
+          "subscribe": [Function],
+        },
+        "unsubscribe": null,
+      },
+    }
+  }
+>
+  <DocumentUpload
```

Comment:
I don't _love_ that it doesn't show the render of DocumentUpload anymore- does it behave differently if you look at component.firstChild in the expect?

Reply:
bummer! It _might_ work if you used the react-testing-library render instead of the enzyme shallow, but it might also be a giant can of worms.